docs/pages/admin-guides/access-controls/resource-rbac.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+Per-resource RBAC enables administrators to control user access on a per-resource basis. Using a granular approach allows you to grant more precise management of user permissions in your environment.
+
+## What is per-resource RBAC?
+
+This capability is particularly useful in complex, multi-tenant environments where user groups have varying levels of access requirements. It allows for precise control over permissions, enabling restricted access to sensitive resources for certain users while granting broader access to others based on their operational needs.
```

Comment:
I would lay out a more explicit definition of per-resource RBAC. Right now, the guide only talks about what it allows, not what it is. I also think we can remove the first heading, since both of these paragraphs can work as introductory text for the guide.

Reply:
Removed initial heading and added a definition